products.js

```diff
@@ -39,31 +45,93 @@ function tunedImageFnodes(nodeToCssMap, coeffImgSize = 2.5, coeffImgHasSrc = 2.5
     }
 
     function imageTitle(fnode) {
-      if (title === undefined) {
-        return 1;
+      const attrs = ['title', 'alt'];
+      for(let i = 0; i < attrs.length; i++){
+        if(fnode.element.getAttribute(attrs[i]) &&
+            (fnode.element.getAttribute(attrs[i]).includes(title) || title.includes(fnode.element.getAttribute(attrs[i])))){
+              return 100 * coeffImgTitle;
```

Comment:
Hmm! So you're propping up the coeffs with multiplication by a constant. Why not factor those into the coeffs themselves? Then I wouldn't be mislead about their relative weights when reading them. (You can and should factor those up even before running the optimizer. The optimizer should be able to reach any number.)